README.md

```diff
@@ -70,13 +70,13 @@ $ git clone https://github.com/google/googletest.git benchmark/googletest
 # Go to the library root directory
 $ cd benchmark
 # Make a build directory to place the build output.
-$ mkdir build && cd build
-# Generate a Makefile with cmake.
-# Use cmake -G <generator> to generate a different file type.
-$ cmake ../
+$ cmake -E make_directory "build"
+# Generate build system files with cmake.
+$ cmake -E chdir "build" cmake -DCMAKE_BUILD_TYPE=Release -DBENCHMARK_ENABLE_LTO=true ../
```

Comment:
ENABLE_LTO is considered an advanced feature which is why it's not in the initial instructions.

I think there's a separate discussion about whether it should just be defaulted to `true` :)

Reply:
Well, as a new user coming to the library I didn't have this impression reading the `README.md`. Instead, I was like: "_well, why didn't you tell me this before I built it without the flag!_" :)

So, perhaps we should change the `README.md` so that it explains what is "advanced" about it and why should one consider not using it. Or we should follow with adding it by default as you mentioned.

Anyway, I'm fine with removing it here if this is what you prefer.

Reply:
filed #1022 to consider changing the default. i think we should remove it from here until we can resolve that so the initial build instructions are as short as possible.

Reply:
OK, I will prepare an update for this pull request.

Reply:
I have fixed this in the new version.